### PR TITLE
Stylish

### PR DESCRIPTION
**Pull request by wrefgtzweve**

### CODE REVIEW

File: lua/mapvote/rtv.lua [hunk 1]
```diff
@@ -113,26 +113,23 @@ function RTV.CanVote(ply)
...
-hook.Add("PlayerSay", "RTV Chat Commands", function(ply, text)
-
-    if table.HasValue(RTV.ChatCommands, string.lower(text)) then
-        RTV.StartVote(ply)
-        return ""
-    end
-
-end)
+hook.Add( "PlayerSay", "RTV Chat Commands", function( ply, text )
+    if not RTV.ChatCommands[tostring( text )] then return end
```
Comment (plally) [inline lua/mapvote/rtv.lua]:
```suggestion
    if not RTV.ChatCommands[string.lower( text )] then return end
```

File: lua/mapvote/rtv.lua [hunk 2]
```diff
@@ -52,53 +56,49 @@ end
...
-function RTV.AddVote(ply)
+function RTV.AddVote( ply )
     ply.RTVVoted = true
-    MsgN(ply:Nick() .. " has voted to Rock the Vote.")
-    PrintMessage(HUD_PRINTTALK,
-                    ply:Nick() .. " has voted to Rock the Vote. (" ..
-                        RTV.GetVoteCount() .. "/" ..
-                        math.Round(RTV.GetPlayerCount() * RTV.PercentPlayersRequired) .. ")")
+    MsgN( ply:Nick() .. " has voted to Rock the Vote." )
+    PrintMessage( HUD_PRINTTALK, ply:Nick() .. " has voted to Rock the Vote. (" .. RTV.GetVoteCount() .. "/" .. math.Round( RTV.GetPlayerCount() * RTV.PercentPlayersRequired ) .. ")" )
```
Comment (plally) [inline lua/mapvote/rtv.lua]: this line is really long

File: lua/mapvote/sv_mapvote.lua [hunk 3]
```diff
@@ -1,227 +1,227 @@
...
+    playCount = util.JSONToTable( file.Read( "mapvote/playcount.txt", "DATA" ) )
 else
     playCount = {}
 end
 
 function CoolDownDoStuff()
     cooldownnum = MapVote.Config.MapsBeforeRevote or 3
 
-    if table.getn(recentmaps) == cooldownnum then table.remove(recentmaps) end
+    if #recentmaps == cooldownnum then table.remove( recentmaps ) end
```
Comment (plally) [inline lua/mapvote/sv_mapvote.lua:45]: should double check these are equivalent

**Review by plally — CHANGES_REQUESTED** [review]

File: lua/autorun/mapvote.lua [hunk 4]
```diff
@@ -32,20 +32,18 @@ MapVote.UPDATE_WIN = 3
...
+    include( "mapvote/config.lua" )
+    include( "mapvote/sv_mapvote.lua" )
+    include( "mapvote/rtv.lua" )
 
-    local files = file.Find("mapvote/integrations/*.lua", "LUA")
-    for _, file in pairs(files) do
-        include("mapvote/integrations/" .. file)
+    local files = file.Find( "mapvote/integrations/*.lua", "LUA" )
+    for _, fil in pairs( files ) do
+        include( "mapvote/integrations/" .. fil )
```
Comment (plally) [inline lua/autorun/mapvote.lua]:
```suggestion
    for _, filename in pairs( files ) do
        include( "mapvote/integrations/" .. filename )
```

File: lua/mapvote/sv_mapvote.lua (hunk 3 above)
Comment (wrefgtzweve) [reply lua/mapvote/sv_mapvote.lua:45]: image

**Pull request merged**